qm.8.md

```diff
@@ -0,0 +1,121 @@
+% QM 8
+
+## NAME
+QM - Setup Containerized environment for running Functional Safety QM (Quality Management) software.
+
+## SYNOPSIS
+This package allow users to setup an environment which prevents applications
+and container tools from interfering with other all other processes on the
+system.
+
+The QM runs its own version of systemd and Podman to isolate not only the
+applications and containers launched by systemd and Podman but systemd and
+Podman themselves.
+
+Software installed into the QM environment under the /usr/lib/qm/rootfs
+directory is automatically isolated from the host. If developers need to
+further isolate their applications from other processes in the QM they should
+use container tools like Podman.
+
+## INSTALL
+
+After you have installed the software package, you need to execute the
+/usr/share/qm/setup script to setup and install the /usr/lib/qm/rootfs
+with packages to run an isolated environment. By default the script installs
```

Comment:
should this be done using RPM dependencies instead of the script?

Reply:
No, I don't believe this would be welcomed into Fedora, it would be triggering an installation procedure during a post install procedure. That is why I kept it separate.

Reply:
oh, it installs these packages in the QM environment. Sorry I had missed this

Reply:
Actually, I see it's not specified, might be worth adding it: `... systemd, and hirte packages in the QM environment, and then enables...`